e2e/utils/TestUtils.ts

```diff
@@ -1,55 +1,71 @@
 import path from 'path';
-import { Application } from 'spectron';
-
-const electronPath = require('electron');
+import os from 'os';
+import fs from 'fs';
+import { Paths, DarwinPaths, LinuxPaths, Win32Paths } from '../../src/utils/paths';
 
 export class TestUtils {
-  public app: Application | undefined;
-
-  public setUp() {
-    this.app = new Application({
-      path:             electronPath as any,
-      args:             [path.join(__dirname, '../../')],
-      chromeDriverArgs: [
-        '--no-sandbox',
-        '--whitelisted-ips=',
-        '--disable-dev-shm-usage',
-      ],
-      webdriverLogPath: './'
-    });
-
-    return this.app.start();
-  }
-
   /**
-   * soft App stop
+   * Create empty default settings to bypass gracefully
+   * FirstPage window.
    */
-  public async tearDown() {
-    if (this.app && this.app.isRunning()) {
-      await this.app.stop();
-    } else {
-      console.log('Something went wrong during stop process.');
+  public createDefaultSettings() {
+    let paths: Paths;
+
+    switch (os.platform()) {
```

Comment:
This code can be replaced like so:
```
type pathsClassType = typeof DarwinPaths|typeof LinuxPaths|typeof Win32Paths;
...
  public createDefaultSettings() {
    const pathInfo: Record<string, pathsClassType> = {
      darwin: DarwinPaths,
      linux:  LinuxPaths,
      win32:  Win32Paths,
    };

    this.createSettingsFile((new pathInfo[os.platform()]()).config);
  }
```

Reply:
And you might be able to just do:
```
public createDefaultSettings() {
    this.createSettingsFile(paths.getPaths().config);
  }
```

but I couldn't get the import of `getPaths` to work.

Reply:
Awesome solution! Thanks for sharing.
I got it working as you have recommended first. 
Not sure the `getPaths` cannot be resolved even being exported...